lib/plugins/aws/deployFunction/index.js

```diff
@@ -20,13 +20,19 @@ class AwsDeployFunction {
     this.hooks = {
       'deploy:function:deploy': () => BbPromise.bind(this)
         .then(this.validate)
+        .then(this.logStatus)
         .then(this.checkIfFunctionExists)
         .then(this.zipFunction)
         .then(this.deployFunction)
         .then(this.cleanup),
     };
   }
 
+  logStatus() {
```

Comment:
Does this need to be in a separate lifecycle? Why not just put it into deployFunction? I'm not sure if we need that message in general, because people specify this command explicitly, so it doesn't provide too much value imho.